src/ui/components/filters/geolocationcomponent.js

```diff
@@ -278,9 +263,9 @@ export default class GeoLocationComponent extends Component {
    * @private
    */
   _saveDataToStorage (query, filter, displayValue, position) {
-    this.core.persistentStorage.set(`${this.name}.${StorageKeys.QUERY}`, query);
+    this.core.persistentStorage.set(`${StorageKeys.QUERY}.${this.name}`, query);
+    this.core.persistentStorage.set(`${StorageKeys.FILTER}.${this.name}`, filter);
```

Comment:
Just wanna make sure I solidify my understanding. This component sets/modifies data in persistent and globalStorage. But, it never actually pulls data from either of them? Looking at the original GeoLocation component (before any of your changes), it looks like it would pull data from globalStorage to set this.filter. That seems unnecessary though since originalFilter isn't a thing for Autocomplete any more. So, the original component didn't actually need to pull from globalStorage at all, is that correct?

Reply:
Yeah, the only thing they needed to pull from globalStorage for was for setting `this.filter` in the constructor. Since we don't need that anymore it's safe to not pull from globalStorage. In the referenced code block we store the filter in the url to be backwards compatible and match previous behavior.